Installs a "locale" service into the "boot" runlevel, to set
a the system locale.

--- runlevels/Makefile.Linux.orig	2009-05-07 15:48:37.075825332 +0000
+++ runlevels/Makefile.Linux	2009-05-07 15:48:52.008557039 +0000
@@ -1,3 +1,3 @@
 SYSINIT+=	devfs dmesg
-BOOT+=		hwclock keymaps modules mtab procfs termencoding
+BOOT+=		hwclock keymaps modules mtab procfs termencoding locale
 SHUTDOWN+=	killprocs mount-ro
--- init.d/Makefile.Linux.orig	2009-05-07 15:55:26.156910483 +0000
+++ init.d/Makefile.Linux	2009-05-07 15:55:35.682853819 +0000
@@ -2,7 +2,7 @@ NET_LO=	net.lo
 
 SRCS+=	devfs.in dmesg.in hwclock.in consolefont.in keymaps.in killprocs.in \
 	modules.in mount-ro.in mtab.in numlock.in procfs.in sysfs.in \
-	termencoding.in
+	termencoding.in locale.in
 
 .SUFFIXES:	.Linux.in
 .Linux.in:
